File: sample/src/main/java/ru/terrakok/cicerone/sample/mvp/bottom/BottomNavigationView.java
package ru.terrakok.cicerone.sample.mvp.bottom;


import moxy.MvpView;
import moxy.viewstate.strategy.AddToEndSingleStrategy;
import moxy.viewstate.strategy.StateStrategyType;

/**
 * Created by terrakok 25.11.16
 */
@StateStrategyType(AddToEndSingleStrategy.class)
public interface BottomNavigationView extends MvpView {
}

File: sample/src/main/java/com/github/terrakok/cicerone/sample/mvp/bottom/BottomNavigationView.kt
package com.github.terrakok.cicerone.sample.mvp.bottom

import moxy.MvpView
import moxy.viewstate.strategy.AddToEndSingleStrategy
import moxy.viewstate.strategy.StateStrategyType

/**
 * Created by terrakok 25.11.16
 */
@StateStrategyType(AddToEndSingleStrategy::class)
interface BottomNavigationView : MvpView